### DISCUSSION TITLE
background: linear-gradient(-90deg, rgb(245, 247, 250) 100%, rgb(255, 255, 255) 100%, rgb(245, 247, 250) 0%); 用unocss 怎么写

### DISCUSSION THREAD

**Discussion by aofong19871029** [unocss/unocss#3229] [Q&A]
### UnoCSS version

0.56.4

### Describe the bug

background: linear-gradient(-90deg, rgb(245, 247, 250) 100%, rgb(255, 255, 255) 100%, rgb(245, 247, 250) 0%); 用unocss 怎么写
还有别的线性渐变和圆形渐变 用uno怎么写

### Reproduction

background: linear-gradient(-90deg, rgb(245, 247, 250) 100%, rgb(255, 255, 255) 100%, rgb(245, 247, 250) 0%); 用unocss 怎么写
还有别的线性渐变和圆形渐变 用uno怎么写

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/unocss/unocss/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't [already an issue](https://github.com/unocss/unocss/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/unocss/unocss/discussions) or join our [Discord Chat Server](https://chat.antfu.me).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by aofong19871029** [comment]
link to https://github.com/unocss/unocss/issues/3227

**Comment by zyyv** [comment]
> Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/unocss/unocss/discussions) or join our [Discord Chat Server](https://chat.antfu.me/).

Please check `Validations` carefully, don't create unnecessary issue.

Answer: [Playground](https://unocss.dev/play/?html=DwEwlgbgBA7gTgQwA5IKZwHwCgpVJWAWgCYAGKACxPIGMAbBAZ0YF4AiAIwHNCvFxUAOwAuhOmEGoEcKIwrJUhANqEAnKRCouAXSgAzOAHsAtsrhcOAClIAaW6QCUuiGARmLl4gFYvdu06hhQ3crb18wmzCnNigAemxgWPAIbCA&config=JYWwDg9gTgLgBAbwFBzgEwKYDNgDsMDCEuOA5gDQpwxQCGuAzltCBlACLBQYDGMwANwwNKqGvSYs2ANVpRg9GAHEoEAK5hKAXzhZVIOAHI1uCDwYNDSUJFiI4YbgwwwAqs6xqANnB16IBsam5gwAtI7CLqFqHt5WSBgAHrbwmFi03qnYeITEZAAUyKgMABbQMDxqMAwAXIhUqADudGBgbHWGjaEMPNwYuHAl3b0Y-bo8E4aivtMRztV1ANoNDk4u7tjehSuowDzEtfWox8c9tF4YdQCMAHQATNMnqDxouB0lMDBgtQD0P8IgG6lH5THYzFZaACU0wAutNxIxmFBWFBDosEZJkWxONw%2BIJhPlodQ6IipFBZPJFCp1GBCXCkFCkEA&css=PQKgBA6gTglgLgUzAYwK4Gc4HsC2YDCAyoWABYJQIA0YAhgHYAmYcUD6AZllDhWOqgAOg7nAB0YAGLcwCAB60cggDYIAXGBDAAUKDBi0mXGADe2sGC704AWgDuCGAHNScDQFYADJ4Dc5sAACtMLKAJ5gggCMLPK2ABR2pPBIcsoAlH4WAEa0yADWTlBYqEw2yFjK3Bpw5LxxAOTllVDoYpSMYgAs3vUZ2gC%2BmsBAA&options=N4IgLgTghgdgzgMwPYQLYgFwKgGzgUwBpxp5k0BhAVzjCVQoGVHNs8iTZEVUAJMVDla4CAXyA)

BTW, we have aligned with tailwind, you should take the initiative to learn how to use it.
